src/common/kqueue.c

```diff
@@ -144,13 +144,14 @@ kqueue(void)
 #endif
 
     kq = calloc(1, sizeof(*kq));
-    if (kq == NULL)
-        return (-1);
+    if (kq == NULL) return (-1);
```

Comment:
Please don't reformat stuff to FR style.

Reply:
done